doc/source/webapp_ui/graph_view/electron.rst

```diff
@@ -2,33 +2,32 @@
 Electron Sidebar
 ===================
 
+The Electron sidebar shows information about a selected task node. Click an electron node in the transport graph to display the sidebar.
+
+
 .. image:: ../images/electron_sidebar.gif
    :align: center
    :width: 5000px
 
-The electron sidebar provides node-level details for a particular workflow dispatch. The sidebar will be shown on the right-hand side when one of the nodes is clicked.
 This sidebar contains the following attributes:
 
-- **Status**
-
-  Shows the status for that particular node/electron.
-
-- **Started - Ended**
-
-  Shows the local time when a workflow dispatch started and when a workflow dispatch ended.
-
-- **Runtime**
-
-  Shows the live runtime for that particular node.
+.. image:: ../../_static/ui_electron_side.png
+   :align: right
 
-- **Input**
```

Comment:
Can we say somewhere in the UI page that all "copy-able" python objects directly copy the bytestring of the object and can be pasted in any running python kernel and unpickled to get the direct python object back. (for example inputs,outputs can be clicked and pasted in jupyter notebook directly from UI without "get_result" from SDK, same for lattice level side bar.)

Reply:
Done.